demo/src/components/IIIFPlayerWrapper.js

```diff
@@ -44,8 +83,33 @@ class IIIFPlayerWrapper extends Component {
             <iiif-explorer
               ref={this.iiifExplorer}
               manifest="https://dlib.indiana.edu/iiif_av/iiif-player-samples/manifest-collections.json"
-              copy-enabled="true"
             />
+            <br />
+            {!this.state.manifestFetched && (
+              <Alert
+                variant="danger"
+                onClose={this.setManifestFetched}
```

Comment:
Yes I agree on the naming of the variable :+1: It sounds better as `showManifestFetchedAlert`. What do you think if we make the alert non-dismissible and hide it on a timeout? Because I think making it non-dismissible and keeping the alert in the UI will not look good from a UX perspective.

Reply:
Sure that could work.  I'm forgetting that this is only on the demo page so we probably don't need to worry about it too much.

Reply:
Yes. This is for only for the GitHub demo page not the actual ReactJS component we export.